packages/models-library/src/models_library/api_schemas_webserver/wallets.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+from ..users import GroupID
+from ..wallets import WalletID, WalletStatus
+
+
+class WalletGet(BaseModel):
+    wallet_id: WalletID
+    name: str
+    description: str | None
+    owner: GroupID
+    thumbnail: str | None
```

Comment:
warning:  both thumbnails and descriptions have been badly misused in the past causing a lot of headackes in models validation. My advice is avoid making them nullable but just not-required, i.e. `WalletGet` description and thumbnails are not required and therefore client should be aware of it. 
When you have no value for those, just do not pass it but avoid using `None` 

See a readable solution in `models_library.utils.pydantic_tools_extensions.FieldNotRequired`